testing/trino-product-tests/src/main/java/io/trino/tests/hive/TestExternalHiveTable.java

```diff
@@ -149,6 +149,38 @@ public void testCreateExternalTableWithInaccessibleSchemaLocation()
         String table = "test_create_external";
         String tableLocation = "/tmp/" + table;
         query(format("CREATE TABLE %s.%s.%s WITH (external_location = '%s') AS SELECT * FROM tpch.tiny.nation", HIVE_CATALOG_WITH_EXTERNAL_WRITES, schema, table, tableLocation));
+
+        String otherSchema = "schema_with_unknown_location";
+        query(format("CREATE SCHEMA %s.%s", HIVE_CATALOG_WITH_EXTERNAL_WRITES, otherSchema));
+
+        String otherTable = "table_with_unknown_schema_location";
+        String otherTableLocation = "/tmp/" + otherTable;
+        query(format("CREATE TABLE %s.%s.%s WITH (external_location = '%s') AS SELECT * FROM tpch.tiny.nation", HIVE_CATALOG_WITH_EXTERNAL_WRITES, otherSchema, otherTable, otherTableLocation));
+    }
+
+    @Test(groups = {HIVE_WITH_EXTERNAL_WRITES, PROFILE_SPECIFIC_TESTS})
+    public void testCreateExternalTableWithExistingPath()
+    {
+        String schema = "schema_with_file_as_location";
```

Comment:
let's create the schema here, because we're not testing with non-existent schema. without schema it should also fail at some point during execution, but that's not the scope here.